### PR TITLE
Add instructions for .DAY

### PR DESCRIPTION
**Pull request by OleMchls**
We need to provide some additional info when providing .DAY domains.

### CODE REVIEW

**Review by SarahCaminiti1 — COMMENTED** [review]
Looks good - only question - is it standard practice to swap between .DAY and .day? Is there a reason we don't stick to .DAY throughout?

**Comment by OleMchls** [comment]
> Looks good - only question - is it standard practice to swap between .DAY and .day? Is there a reason we don't stick to .DAY throughout?

@SarahCaminiti1 I just copied .DE 🤷  I've a personal preference for .TLD but don't mind either way in the end. What do you prefer?

**Comment by weppos** [comment]
> > Looks good - only question - is it standard practice to swap between .DAY and .day? Is there a reason we don't stick to .DAY throughout?
> 
> @SarahCaminiti1 I just copied .DE 🤷 I've a personal preference for .TLD but don't mind either way in the end. What do you prefer?

I second it. I'd prefer we stick with the UPPERCASE version consistently when referring to a specific .TLD.

File: content/articles/domains-day.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+This article explains the requirements and special procedures for .DAY domain names.
+
+
+## Registering a .DAY domain {#registering}
+
+.day is a secured namespace, meaning that websites using a .day domain require an SSL certificate to work; .day websites can only be accessed by web browsers using HTTPS through an encrypted and secured connection.
+
+You can obtain a certificate for your .day domain through DNSimple. To learn more, visit [SSL Certificates](/articles/ssl-certificates).
```
Comment (weppos) [inline content/articles/domains-day.markdown:22]: What about a call to action to the site.

```suggestion
You can [obtain a certificate for your .day domain](https://dnsimple.com/ssl-certificates) through DNSimple. To learn more, visit [SSL Certificates](/articles/ssl-certificates).
```

Comment (OleMchls) [reply content/articles/domains-day.markdown]: Done in [4217023](https://github.com/dnsimple/dnsimple-support/pull/949/commits/42170232600f7dd4f3055c089b35c01e8fd1ffc4)

**Pull request merged**